packages/parser/src/types/schema.ts

```diff
@@ -171,6 +174,7 @@ export type {
   SnsRecord,
   SqsEvent,
   SqsRecord,
+  TransferFamilyEvent,
```

Comment:
Please add this type [here](https://github.com/aws-powertools/powertools-lambda-typescript/blob/66364b5f9c74df7eefc695aab788c1f4008e7250/packages/parser/src/types/index.ts).